docs/core/interactive-learning.rst

```diff
@@ -30,14 +30,14 @@ Run the following command to start interactive learning:
    python3 -m rasa_sdk --actions actions&
 
    rasa interactive \
-     -o models \
+     --out models \
      -d domain.yml \
      -c policy_config.yml \
      --endpoints endpoints.yml
 
 The first command starts the action server (see :ref:`custom-actions`).
 
-The second command trains starts interactive learning mode
+The second command trains a model and starts interactive learning mode.
```

Comment:
I think we should change the default example to use an existing model, since that makes way more sense that training on startup